### DISCUSSION TITLE
How to actually enable the mirror?

### DISCUSSION THREAD

**Discussion by KaranAhlawat** [fardinkamal62/fedora-mirror-measure#6] [Q&A] [answered]
Hello! Thank you for the tool, it seems very useful for a large number of people.

I've used the tool and have figured out the mirror I'd like to use. The question I have is, how do I actually get DNF to use the mirror I want? Looking on the internet did not really provide any answers I thought were useful. I didn't think this warranted an issue, so I'm starting a Discussion instead.

Thank you

**Comment by fardinkamal62** [comment, marked as answer]
**Thank you!**

If you've figured out which repo you want to use, copy the url like I did in this screenshot

![Screenshot_20230328_112134](https://user-images.githubusercontent.com/70953546/228136116-f7825b83-0ed3-4eb1-b7d8-a9cb91986a9f.png)


Now open `/etc/yum.repos.d/fedora.repo` with any text editor as root and you'll see something kinda similar

![image](https://user-images.githubusercontent.com/70953546/228136493-27ff8c8d-32f3-4cea-9f43-7e325bed9547.png)

Comment the `metalink` line; it checks & selects repo automatically
and uncomment `baseurl`; it hard codes repo

![image](https://user-images.githubusercontent.com/70953546/228136871-511fde01-c3e1-404b-9b02-177b79ea11a7.png)

In the baseurl paste your `https://{url}/fedora/linux` and rest will be same
In the example my repo url was `https://codingflyboy.mm.fcix.net/fedora/linux/` and the rest `/releases/$releasever/Everything/$basearch/os/` will be same

Here I've changed `/etc/yum.repos.d/fedora.repo`, you can change other files of `/etc/yum.repos.d` too. Just check if the mirror has the resources or not.
Hope it's clear

**Reply by KaranAhlawat** [reply]
That sounds clear and to the point, thanks again!
I'll try the solution and if it works mark it as an answer.
